cfgov/searchgov/jinja2/searchgov/index.html

```diff
@@ -66,9 +66,10 @@ <h1>{{ results_header if query else search_header }}</h1>
             {% set en_results = _('Results ') %}
             {% set en_of = _(' of ') %}
             {% set en_for = _(' for ') %}
+            {% set en_count = count if count < 999 else _(' 1,000+ ') %}
```

Comment:
Can we set the string here to '1000+' and the translation to '+1000'? The spaces are being provided by the of/for bits so this will be duplicating them (I think it visually collapses but is still present in the html)

Reply:
Also I updated against main so that snyk would be happy, so pull first before changing 🕺

Reply:
Good catch, thanks. Fixed in f7c0cbf.